gitlab_runner/metadata.csv

```diff
@@ -3,12 +3,18 @@ gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_buck
 gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_sum,gauge,,request,second,Sum of Docker machine creation time,0,gitlab_runner,sum docker machine creation time
 gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_count,gauge,,request,second,Count of Docker machine creation time,0,gitlab_runner,count docker machine creation time
 gitlab_runner.ci_docker_machines_provider_machine_states,gauge,,request,second,The current number of machines per state in this provider,0,gitlab_runner,total docker machines per state
+gitlab_runner.ci_runner_builds,gauge,,,,The current number of running builds.,0,gitlab_runner,current running builds
 gitlab_runner.ci_runner_errors,counter,,request,second,The number of caught errors,0,gitlab_runner,errors
 gitlab_runner.ci_runner_version_info,gauge,,request,,A metric with a constant '1' value labeled by different build stats fields,0,gitlab_runner,version info
 gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_bucket,gauge,,request,second,Histogram of SSH Docker machine creation time,0,gitlab_runner,ssh docker machine creation time
 gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_sum,gauge,,request,second,Sum of SSH Docker machine creation time,0,gitlab_runner,sum ssh docker machine creation time
 gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_count,gauge,,request,second,Count of SSH Docker machine creation time,0,gitlab_runner,count ssh docker machine creation time
 gitlab_runner.ci_ssh_docker_machines_provider_machine_states,gauge,,request,second,The current number of machines per state in this provider,0,gitlab_runner,total ssh docker machines per state
+gitlab_runner.gitlab_runner_autoscaling_machine_creation_duration_seconds,gauge,,request,second,Histogram of Docker machine creation time,0,gitlab_runner,docker machine creation time
```

Comment:
Should we specify in the description that is from a specific version? (and also in the old ones that only appear before a specific version)